packages/thirdweb/src/react/web/hooks/useSendTransaction.tsx

```diff
@@ -1,5 +1,7 @@
 import { useContext, useState } from "react";
 import type { ThirdwebClient } from "../../../client/client.js";
+import { useActiveWallet } from "../../../exports/react-native.js";
```

Comment:
Is this the right import (from react-native exports)?

Reply:
definitely not

Reply:
aah - auto imports 😞

Reply:
Fixed now